src/pages/lesson-page/lesson-page.tsx

```diff
@@ -1,28 +1,87 @@
-import classNames from 'classnames';
-import styles from './lesson-page.module.scss';
+import { ChevronLeftIcon } from '@radix-ui/react-icons';
 import commonStyles from '@styles/common-styles.module.scss';
+import classNames from 'classnames';
+import { format } from 'date-fns';
+import { useState } from 'react';
 import { useParams } from 'react-router-dom';
-import { RouteParams } from '/src/router/config';
+import styles from './lesson-page.module.scss';
 import { useAvailability, useLessonBySlug } from '/src/api/api-hooks';
-import { useState } from 'react';
 import { Calendar } from '/src/components/calendar/calendar';
+import { LessonDetails } from '/src/components/lesson-details/lesson-details';
+import { RouteParams } from '/src/router/config';
+import { HourButtons } from '/src/components/hour-buttons/hour-buttons';
 
-export interface LessonPageProps {
-    className?: string;
-}
-
-export const LessonPage = ({ className }: LessonPageProps) => {
+export const LessonPage: React.FC = () => {
     const { slug } = useParams<RouteParams['/lesson/:slug']>();
     const { data } = useLessonBySlug(slug);
```

Comment:
lesson data probably should have isLoading, the lesson data might not be here yet